x-pack/test/cases_api_integration/security_and_spaces/tests/trial/cases/assignees.ts

```diff
@@ -189,6 +189,30 @@ export default ({ getService }: FtrProviderContext): void => {
         });
       });
 
+      it('filters cases using an assignee uid with a colon in it', async () => {
```

Comment:
filter by assignees is only available to trial?

Reply:
Good point. It's the suggestion api that is not in the basic license. This test doesn't use that so I'll move it to the `find_cases` file.

Reply:
Actually never mind you can't assign users unless you have a platinum license either. So I think this test should probably stay within the trial folder.

Reply:
That's correct, assigning and using the suggestion API requires a platinum license. Filtering by assignee doesn't require a platinum license. The issue with moving this test to the basic directory is that I wouldn't be able to test retrieving the case with weird characters in the assignee uid because I wouldn't be able to assign a user with a weird uid.

Reply:
> The issue with moving this test to the basic directory is that I wouldn't be able to test retrieving the case with weird characters in the assignee uid because I wouldn't be able to assign a user with a weird uid.

yy that's fine, you can leave it 🙌